WORKSPACE

```diff
@@ -42,7 +42,7 @@ bazel_skylib_workspace()
 
 load("@build_bazel_rules_nodejs//:index.bzl", "check_bazel_version", "node_repositories", "yarn_install")
 
-# The minimum bazel version to use with this repo is v3.1.0.
+# The minimum bazel version to use with this repo is v4.0.0.
```

Comment:
IMO we can just remove this comment, the code is pretty self-explanatory.

Reply:
good idea